doc/.vuepress/config.js

```diff
@@ -14,13 +14,13 @@ module.exports = {
             {
               collapsable: false,
               title: 'Guide',
-              children: ['', 'intro', 'shell', 'package-manager', 'ffi', 'build', 'profiler', 'srfi', 'core-prelude', 'r7rs', 'bootstrap', 'nix']
+                children: ['', 'intro', 'getting-started', 'shell', 'package-manager', 'ffi', 'build', 'profiler', 'bootstrap', 'nix', 'r7rs', 'srfi', 'core-prelude']
```

Comment:
Minor nit: why the indentation here?

Reply:
hrm, probably some tab involved.

Reply:
no, it's just emacs wanting to indent this way.